cider-velox/src/ArrowConvertorUtils.cpp

```diff
@@ -20,6 +20,7 @@
  */
 
 #include "ArrowConvertorUtils.h"
+#include <cstdint>
```

Comment:
Don't include unnecessary header.  Same as below.

Reply:
Besides, ```ArrowConvertor``` is not in use actually, is it necessary to change it ?

Reply:
Fixed, I think we can refactor it first,  then make changes later